### PR TITLE
fix(search): Handle search errors and escape *

### LINKED ISSUE #890: Wildcard causing browser crash
**Describe the bug**
Putting a asterisk (wildcard) in the search box causes the browser to crash.

**What version of Overseerr are you running?**
1.19.1

**To Reproduce**

1. Go to Discover
2. Click on Search Movies & TV
3. Type *
4. Have a cup of tea

**Expected behavior**
Nothing to show in results, or at the very least not crash

[**Screenshot**](https://i.imgur.com/MoEbktR.png)

**Desktop (please complete the following information):**

- OS: MacOS
- Browser Chrome
- Version 88

**Additional context**

### PR DESCRIPTION
**Pull request by ankarhem**
#### Description
escapes * and correctly handles when search gives error

#### Todos

- [x] Sucessfully builds `yarn build`

#### Issues Fixed or Closed by this PR

- Fixes #890

### CODE REVIEW

File: src/components/Search/index.tsx [hunk 1]
```diff
@@ -53,7 +53,7 @@ const Search: React.FC = () => {
   };
 
   if (error) {
-    return <div>{error}</div>;
+    console.error(error);
```
Comment (sct) [inline src/components/Search/index.tsx:56]: Is it intentional to have a console error here?

Let's render the `<Error />` component instead?

Comment (sct) [reply src/components/Search/index.tsx:56]: Oh, I see. You don't want to stop rendering in the case of an error. I still wonder if it's necessary?

Comment (ankarhem) [reply src/components/Search/index.tsx]: Uses the error component instead now. Rebased.

**Review by sct — APPROVED** [review]
💯

**Pull request merged**